safe/gis/vector/summary_5_multi_exposure.py

```diff
@@ -157,6 +158,7 @@ def multi_exposure_analysis_summary(analysis, intermediate_analysis):
     analysis.commitChanges()
     analysis.keywords['title'] = layer_purpose_analysis_impacted['name']
     analysis.keywords['layer_purpose'] = layer_purpose_analysis_impacted['key']
+    analysis.keywords['exposure_keywords'] = MULTI_EXPOSURE_ANALYSIS_FLAG
```

Comment:
Why do you set this flag here?
`exposure_keywords` must be keywords from the exposure layer.

Reply:
I'm about to ask this thing. because it has multiple exposures, could we set the exposure keywords with list of exposure key or better to create new keywords which tell us whether that layer is coming from multi exposure or not?

Reply:
Yes I understand.
Hum, it would be better to have a dictionary, pseudocode:
```
{
    roads exposure key: roads keywords list,
    population exposure key: population keywords list,
}
```
No ?

Reply:
so @ismailsunni suggest me to put that flag as an `extra_keywords`. I think I will go with that

Reply:
You will go with the flag you mean?
Sorry, I'm confused, where is the suggestion? Why `extra_keywords` ? So you drop `exposure_keywords` ?

Reply:
If we go with flag in `extra_keywords`, `exposure_keywords` will be just empty as it is. I prefer with flag in `extra_keywords` because I think we will introduce new standard in filling the `exposure_keywords` if we use the `extra_keywords` because single IF already has theirs.

Reply:
exposure_keywords must stay the same across IF. So it's either not present or with keywords from exposures. So yes, don't put that flag here I think.

Reply:
> Sorry, I'm confused, where is the suggestion?

in private chat

Reply:
it would be nice to be on gitter ;-)